task.ps1

```diff
@@ -1 +1,15 @@
-# Write your code here
+$VirtualMachine = Get-AzVM `
+    -ResourceGroupName "mate-resources-ukwest" `
+    -Name "mate-vm"
+Remove-AzVMDataDisk `
+    -VM $VirtualMachine `
+    -Name "mate-task-2-disk"
+Update-AzVM `
+    -ResourceGroupName "mate-resources-ukwest" `
+    -VM $VirtualMachine
+
+$allDisks = Get-AzDisk -ResourceGroupName "mate-resources-ukwest"
+
+$unattachedDisks = $allDisks | Where-Object { $_.ManagedBy -eq $null -or $_.DiskState -eq 'Unattached' }
+
+$unattachedDisks | Select-Object -Property Id, Name, DiskSizeGB, DiskState | ConvertTo-Json | Set-Content -Path "./result.json"
```

Comment:
![image](https://github.com/user-attachments/assets/d1d7ff07-557d-4d18-9100-a4217f7ebe07)
Add Eof(End-of-file), need to add blank line at the end